code/go/0chain.net/blobbercore/challenge/worker.go

```diff
@@ -14,11 +14,18 @@ func SetupWorkers(ctx context.Context) {
 	go startCommitProcessed(ctx)
 }
 
+var (
+	waitToProcess = make(chan *ChallengeEntity)
+	waitToCommit  = make(chan *ChallengeEntity)
+)
+
 func startCommitProcessed(ctx context.Context) {
 	for {
 		select {
 		case <-ctx.Done():
 			return
+		case next := <-waitToCommit:
+			commitChallenge(next)
```

Comment:
do we still needs to be called from `commitProcessed` ?

Reply:
@sum-it08  Yes. it is keep challenge will be lost to commit even blobber crashes and restarts.